spec/cucumber/step_match_search_spec.rb

```diff
@@ -111,7 +111,7 @@ module Cucumber
         end
       end
 
-      # TODO: remove this - it's ... redundant
+      #  TODO: remove this - it's ... redundant
       # http://railsforum.com/viewtopic.php?pid=93881
```

Comment:
Please, can you make sure that there is only one space before the `TODO` word? 

The misleading bit is that as first character there is a NON-BREAK SPACE utf-8 character (https://en.wikipedia.org/wiki/Non-breaking_space), which should be removed. Please can you fix also the lines above, there are many with this problem.